scripts/cargo-install-all.sh

```diff
@@ -110,13 +110,19 @@ done
   set -x
   # shellcheck disable=SC2086 # Don't want to double quote $rust_version
   $cargo $maybeRustVersion build $maybeReleaseFlag "${binArgs[@]}"
+  $cargo install spl-token-cli
```

Comment:
```
cargo install spl-token-cli --root "$installDir"
```

Reply:
well, after the `mkdir -p "$installDir/bin"`

Reply:
Ah, nice!